src/battle-scene-interface.ts

```diff
@@ -0,0 +1,69 @@
+	animating: boolean = false;
+	acceleration: number = NaN;
+	gen: number = NaN;
+	activeCount: number = NaN;
+	numericId: number = NaN;
+	timeOffset: number = NaN;
+	interruptionCount: number = NaN;
+	messagebarOpen: boolean = false;
+
+	abilityActivateAnim(pokemon: Pokemon, result: string): void { }
+	addPokemonSprite(pokemon: Pokemon): PokemonSprite { return null!; }
+	addSideCondition(siden: number, id: ID, instant?: boolean | undefined): void { }
+	animationOff(): void { }
+	animationOn(): void { }
+	closeMessagebar(): void { }
+	damageAnim(pokemon: Pokemon, damage: string | number): void { }
+	destroy(): void { }
+	finishAnimations(): JQuery.Promise<JQuery<HTMLElement>, any, any> | undefined { return void(0); }
+	healAnim(pokemon: Pokemon, damage: string | number): void { }
+	hideJoinButtons(): void { }
+	incrementTurn(): void { }
+	log(html: string, preempt?: boolean | undefined): void { }
+	message(message: string, hiddenMessage?: string | undefined): void { }
+	pause(): void { }
+	preemptCatchup(): void { }
+	removeSideCondition(siden: number, id: ID): void { }
+	reset(): void { }
+	resultAnim(pokemon: Pokemon, result: string, type: "bad" | "good" | "neutral" | "par" | "psn" | "frz" | "slp" | "brn"): void { }
+	resume(): void { }
+	runMoveAnim(moveid: ID, participants: Pokemon[]): void { }
+	runOtherAnim(moveid: ID, participants: Pokemon[]): void { }
+	runPrepareAnim(moveid: ID, attacker: Pokemon, defender: Pokemon): void { }
+	runResidualAnim(moveid: ID, pokemon: Pokemon): void { }
+	runStatusAnim(moveid: ID, participants: Pokemon[]): void { }
+	soundStart(): void { }
+	soundStop(): void { }
+	startAnimations(): void { }
+	teamPreview(): void { }
+	teamPreviewEnd(): void { }
+	updateGen(): void { }
+	updateSidebar(side: Side): void { }
+	updateSidebars(): void { }
+	updateStatbars(): void { }
+	updateWeather(instant?: boolean | undefined): void { }
+	upkeepWeather(): void { }
+	wait(time: number): void { }
+	setFrameHTML(html: any): void { }
+	setControlsHTML(html: any): void { }
+
+	removeEffect(pokemon: Pokemon, id: ID, instant?: boolean) { }
+	addEffect(pokemon: Pokemon, id: ID, instant?: boolean) { }
+	animSummon(pokemon: Pokemon, slot: number, instant?: boolean) { }
+	animUnsummon(pokemon: Pokemon, instant?: boolean) { }
+	animDragIn(pokemon: Pokemon, slot: number) { }
+	animDragOut(pokemon: Pokemon) { }
+	updateStatbar(pokemon: Pokemon, updatePrevhp?: boolean, updateHp?: boolean) { }
+	updateStatbarIfExists(pokemon: Pokemon, updatePrevhp?: boolean, updateHp?: boolean) { }
+	animTransform(pokemon: Pokemon, isCustomAnim?: boolean, isPermanent?: boolean) { }
+	destroyPokemonSprite(pokemon: Pokemon) { }
+	clearEffects(pokemon: Pokemon) { }
+	removeTransform(pokemon: Pokemon) { }
+	animFaint(pokemon: Pokemon) { }
+	animReset(pokemon: Pokemon) { }
+	anim(pokemon: Pokemon, end: ScenePos, transition?: string) { }
+	beforeMove(pokemon: Pokemon) { }
+	afterMove(pokemon: Pokemon) { }
+	updateSpritesForSide(side: Side) { }
+}
```

Comment:
![image](https://user-images.githubusercontent.com/551184/46910299-cb758f00-cf07-11e8-9746-3a37380e6c92.png)

Reply:
(You're missing a trailing linebreak)